plugin/src/main/java/net/dzikoysk/funnyguilds/data/util/InvitationList.java

```diff
@@ -204,7 +203,7 @@ public UUID getFor() {
 
         @Nullable
         private Guild wrapFrom() {
-            return GuildUtils.getByUUID(from);
+            return FunnyGuilds.getInstance().getGuildManager().getGuild(from).getOrNull();
```

Comment:
Nie lepiej tu zwracać po prostu Option<Guild> zamiast to cringowo odpakowywać i oznaczać jako @Nullable